sympy/core/relational.py

```diff
@@ -1164,6 +1203,39 @@ def _eval_fuzzy_relation(cls, lhs, rhs):
     def weak(self):
         return Ge(*self.args)
 
+    def _eval_simplify(self, **kwargs):
+        # standard simplify
+        eorg = e = super()._eval_simplify(**kwargs)
+        if isinstance(e, StrictLessThan):
+            e = e.reversed
+        if not isinstance(e, StrictGreaterThan):
+            return e
+        from .expr import Expr
+        if not isinstance(e.lhs, Expr) or not isinstance(e.rhs, Expr):
+            return e
+        from sympy.functions import ceiling, floor
+        lhs_is_int = e.lhs.is_integer
+        rhs_is_int = e.rhs.is_integer
+        r = e
+        if lhs_is_int is True:
+            if rhs_is_int is True:
+                if e.lhs.is_number:
+                    r = Ge(e.lhs - 1, e.rhs)
```

Comment:
I guess it's subjective, but I wonder if it would be better to put use `e.rhs <= e.lhs - 1` so that the number is on the right-hand side.

Reply:
The reason is that this is a `Ge` to start with, so returns it like that. Then, there is a canonical call anyway further down.